networkx/generators/geometric.py

```diff
@@ -279,14 +432,19 @@ def geographical_threshold_graph(n, theta, alpha=2, dim=2, pos=None,
     nx.set_node_attributes(G, 'weight', weight)
     nx.set_node_attributes(G, 'pos', pos)
 
+    #if p_dist not supplied, use default r^alpha
+    if p_dist == None:
+        def p_dist(r):
+            return r**-alpha
+
```

Comment:
I've kept the alpha parameter for this default function to allow for backward comp. It would be cleaning to just remove the alpha parameter all together and have a single default function, but not sure a way to do that without breaking backwards comp.